run-checks

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
```

Comment:
I don't mind considering adding a script to help running all checks, however, I'd prefer to do it in a separate PR with its own discussion. So let's remove this from this PR.

If we do add a script for this, let's keep it ultra-simple with no fancy Bash logic. I agree with @mwhudson's lightning talk about "stop writing Bash" :-), as it's error prone and hard to maintain. I'd probably suggest a `Makefile` with a `make check` target.

I personally don't mind typing `go test ./...`, but I'd be interested to see what other Gophers do/recommend.

Reply:
Oops! Accidentally committed this file (shouldn't be here)